manifoldco/ui
# added mf-input

https://app.zenhub.com/workspaces/engineering-56990653b8516b1b4867e8b2/issues/manifoldco/engineering/7570

## Reason for change

- Added `mf-input`, a generic text input field.
- Used in `mf-slider` (soon to be irrelevant) and for the search/filter field for the service grid

<img width="808" alt="Screen Shot 2019-03-20 at 5 36 37 PM" src="https://user-images.githubusercontent.com/10498708/54717383-ce1f8480-4b36-11e9-9a91-ca2c56778508.png">
<img width="793" alt="Screen Shot 2019-03-20 at 5 36 56 PM" src="https://user-images.githubusercontent.com/10498708/54717385-ce1f8480-4b36-11e9-8a37-e587fd92e247.png">

<img width="329" alt="Screen Shot 2019-03-19 at 10 15 08 PM" src="https://user-images.githubusercontent.com/10498708/54652132-7c262280-4a94-11e9-83ff-4dbd506fd990.png">


## Testing


- Check the styling and functionality of the service grid search/filter feature.
- Check the styling of the custom plan selector slider field type. (We might be getting rid of the slider soon so this is not as important.)
